src/Domain/Person.php

```diff
@@ -413,4 +413,47 @@ public function makeNotCompromisedValidator(): Assert\NotCompromisedPasswordVali
         $notCompromisedValidator = new Assert\NotCompromisedPasswordValidator($httpClient);
         return $notCompromisedValidator;
     }
+
+    public function getStripeCustomerParams(\DateTimeImmutable $passwordAddedAt = null): array
```

Comment:
I think the parameter might unnecessary, as the existing property `$this->email_address_verified` is essentially the same thing as passwordAddedAt, although that won't be true in future if we allow people with passwords set to edit their details without verifying email address.

If we have the param probably worth adding an assertion that if its not null there is a password set on the object.

Reply:
Yep I think you're right, good spot. Removed in e630eb1cfa94f9afb5fda4030dfce7372f2d4e9b